### PR TITLE
docs(cawemo): add notice 109 and changes to the download page

### PR DESCRIPTION
**Pull request by JacobiCamunda**
Part of https://github.com/camunda/cawemo/issues/11774

### CODE REVIEW

File: security/content/notices.md [hunk 1]
```diff
@@ -15,6 +15,50 @@ releases of the community platform.
...
+
+**Product affected:**
+
+Cawemo On-Premises
+
+**Impact:**
+
+The version of `libcrypto3` and `libssl3` shipped with `cawemo-restapi` was affected by the following vulnerability:
+
+- https://nvd.nist.gov/vuln/detail/CVE-2024-2511
```
Comment (wollefitz) [inline security/content/notices.md:30]: This can be merged with the report of the same CVE below

File: security/content/notices.md [hunk 2]
```diff
@@ -15,6 +15,46 @@ releases of the community platform.
...
+
+**Publication Date: May 13th, 2024**
+
+**Product affected:**
+
+Cawemo On-Premises
+
+**Impact:**
+
+The versions of `libcrypto3` and `libssl3` shipped with `cawemo-restapi` and `cawemo-websockets` were affected by the following vulnerability:
```
Comment (wollefitz) [inline security/content/notices.md:28]: ❌ This should also list `cawemo-webapp`

**Review by wollefitz — CHANGES_REQUESTED** [review]

**Review by wollefitz — COMMENTED** [review]
Additionally, `openssl` is affected by CVE-2024-2511 shipped via `cawemo-websockets`

**Comment by JacobiCamunda** [comment]
@jfriedenstab, requesting your final review because Wolfgang is for the rest of the day and I want to get this merged and released today.

**Review by jfriedenstab — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**